CriteoPublisherSdk/Sources/CRConstants.h

```diff
@@ -22,4 +22,6 @@
 
 #define CRITEO_PUBLISHER_SDK_VERSION @"3.10.0"
 
+#define CRITEO_AD_NETWORK_IDENTIFIER @"hs6bdukanm.skadnetwork"
```

Comment:
Emphasizing one of the recommendations that I made internally: should we have this in a CaC or returned by CDB instead? If we revoke this ID and generate a new one (who knows for what reason), this would prevent us from having to update the SDK.

Reply:
In fact, following up on my API feedback, I think the SDK shouldn't even know about this ID (except may be to define it in the `Info.plist`). The SDK should send the full list of IDs defined in the `Info.plist` to the CDB as the campaign should be attributed to either of the DSPs represented by that ID.